src/Generators/AzureFunctions/DurableFunction.cs

```diff
@@ -22,21 +22,24 @@ public class DurableFunction
         public DurableFunctionKind Kind { get; }
         public TypedParameter Parameter { get; }
         public string ReturnType { get; }
+        public bool ReturnsVoid { get; }
 
         public DurableFunction(
             string fullTypeName,
             string name,
             DurableFunctionKind kind,
             TypedParameter parameter,
-            ITypeSymbol returnType,
+            ITypeSymbol? returnType,
+            bool returnsVoid,
```

Comment:
Isn't adding a new member to a constructor without either default value or a separate constructor with the same types technically a breaking change? I feel like this constructor shouldn't be called by a user, but it is public so we may have to worry about that.